docs/croissant-spec.md

```diff
@@ -263,14 +263,40 @@ corresponding properties must be defined using `subField`.
 
 A field may have more than a single assigned `dataType`, in which case at least
 one must inform about the expected type of data (eg: `sc:Text`), while other
-types inform about the semantic being used.
+types inform about the semantic being used, possibly semantics with ML meaning.
 
 **range**:	[sc:Text](https://schema.org/Text), [sc:URL](https://schema.org/URL)
 
 **domain**:	[Field](#field)
 
 
-In the following example, the `url` field is expected to be a URL , which
+Supported types with ML meaning:
+
+| `dataType` | Usage |
+| -------- | ----- |
+| [**`sc:ImageObject`**](https://schema.org/ImageObject) | Describes a field containing the content of an image (pixels). |
+| [**`ml:BoundingBox`**](http://mlcommons.org/schema/BoundingBox) | Describes a bounding box. |
+| [**`sc:name`**](https://schema.org/name) | Describes a field which can be used as a human-friendly label. |
```

Comment:
Should this be in `Supported types with no ML meaning`?

Reply:
Merged everything in "Supported known types", clearer in the end IMO.